common/rbnf/sr_Latn.xml

```diff
@@ -115,10 +115,10 @@ x.x: << koma >>;
 1000000000000000000: =#,##0=;
 %%ordi:
 0: i;
-1: ‘ i =%spellout-ordinal=;
+1: ' i =%spellout-ordinal=;
```

Comment:
I don't know where the left hand side came from - maybe by hand. 
However, this line is actually converted from:

- https://github.com/unicode-org/cldr/blob/6a57fe5413950e6ed03042de179856c47074a28d/common/rbnf/sr.xml#L117-L118

which shows `'`